class/defaults.yml

```diff
@@ -1,6 +1,15 @@
 parameters:
   thanos:
     namespace: syn-thanos
+    thanos_mixin_version:
+      "1.18": "79d8cfdc1a00f8a96475d5d1ff1a852b184b146e"
+      "1.19": "79d8cfdc1a00f8a96475d5d1ff1a852b184b146e"
+      "1.20": "09b36547e5ed61a32a309648a8913bd02c08d3cc"
+      "1.21": "ff363498fc95cfe17de894d7237bcf38bdd0bc36"
+      "1.22": "ff363498fc95cfe17de894d7237bcf38bdd0bc36"
```

Comment:
Can you
a) add comments describing which branch/tag those commit hashes refer to, or
b) use branch/tag names instead of commit hashes?